xbmc/LangInfo.cpp

```diff
@@ -300,6 +302,7 @@ void CLangInfo::CRegion::SetGlobalLocale()
     strLocale = lcl.name();
     current_locale = current_locale.combine< std::collate<wchar_t> >(lcl);
     current_locale = current_locale.combine< std::ctype<wchar_t> >(lcl);
+    current_locale = current_locale.combine< std::time<wchar_t> >(lcl);
```

Comment:
std::time seems to not support a type definition via <>  every build on any platform crashes right now.